_compendium_entries/2020-09-25.md

```diff
@@ -20,5 +20,5 @@ Bonny
 1. [Building Your First Web Page - (seriously this has everything)](https://learn.shayhowe.com/html-css/building-your-first-web-page){: target="_blank" .entry-link}
 2. [Hosting your new website for free (Tutorial)](https://medium.com/@svinkle/publish-and-share-your-own-website-for-free-with-github-2eff049a1cb5){: target="_blank" .entry-link}
 3. [Visualizations for Data Structures and Algorithms in CS](https://www.cs.usfca.edu/~galles/visualization/Algorithms.html){: target="_blank" .entry-link}
-4. The Second Kanye Fact™ - did you know on september 15th, kanye west started talking about how egregious the contracts that major labels place on young individuals are, and was infuriated by this. on september 16th, a day later, he proceeded to tweet out each page of his contracts, but not before failing to do that, by accidentally tweeting out the file name instead of the actual file. later on in the day, he tweeted a video of him engaging in questionable actions upon a grammy award (serves them right for the amount of snubs they've given out especially in 2013), and then got banned from twitter for 12 hours for revealing personal information about a white supremacist. (courtesy of rohan v suri)
+4. The Second Kanye Fact™ - did you know on september 15th, kanye west started talking about how egregious the contracts that major labels place on young individuals are, and was infuriated by this. on september 16th, a day later, he proceeded to tweet out each page of his contracts, but not before failing to do that, by accidentally tweeting out the file name instead of the actual file. later on in the day, he tweeted a video of him engaging in questionable actions upon a grammy award (serves them right for the amount of snubs they've given out especially in 2013), and then got banned from twitter for 12 hours for revealing personal information about a white supremacist. (courtesy of jonathan oppenheimer)
```

Comment:
```suggestion
4. The Second Kanye Fact™ - did you know on september 15th, kanye west started talking about how egregious the contracts that major labels place on young individuals are, and was infuriated by this. on september 16th, a day later, he proceeded to tweet out each page of his contracts, but not before failing to do that, by accidentally tweeting out the file name instead of the actual file. later on in the day, he tweeted a video of him engaging in questionable actions upon a grammy award (serves them right for the amount of snubs they've given out especially in 2013), and then got banned from twitter for 12 hours for revealing personal information about a white supremacist. (courtesy of rohan v surir)
```